launch/dynamo-run/src/subprocess/sglang_inc.py

```diff
@@ -29,14 +31,17 @@
 DEFAULT_ENDPOINT = "dyn://dynamo.backend.generate"
 DEFAULT_MODEL = "Qwen/Qwen2.5-0.5B-Instruct"
 
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Should this be DEBUG by default?

Reply:
It turns out not to matter, because the stdout/stderr of this sub-process goes to Rust, where it gets sent to the `tracing` module. We still control it via `DYN_LOG`.

Reply:
Interesting, works for me then 🤷